Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -732,6 +732,18 @@ class Comment: Operation {
     }
 }
 
+class BeginCodeString: Operation {
```

Comment:
Can you add a small comment here about what this operation does? I.e. that it turns its body into a string literal containing the lifted code, which is then particularly suited to pass to `eval()`

Reply:
done